frontend/css/listens-page.less

```diff
@@ -66,6 +66,12 @@
     }
   }
 
+  .dropdown-toggle {
+    &::after {
+      display: none !important;
+    }
+  }
```

Comment:
Would it make sense to make this a more general style in order to replicate our current design?
For example on the playlist page the caret is now visible:
![image](https://github.com/user-attachments/assets/6710e1cd-3553-4f5f-ae73-f64994276ae7)


I suppose there are good reasons for BS to want to show carets for all dropdowns, so a decision needs to be made, but in general i would aim to replicate our current look and only afterwards evaluate if we want this change